packages/teams-js/test/private/copilot.spec.ts

```diff
@@ -384,4 +387,24 @@ describe('copilot', () => {
       });
     });
   });
+
+  describe('copilot.customTelemetry', () => {
+    describe('sendCustomTelemetryData', () => {
```

Comment:
>describe [](http://example.com/codeflow?start=4&length=8)

Should there be tests to validate that the timestamp is recorded and sent properly when it is not provided? And that it is passed-on when it is provided?

Reply:
Yes, there should. I tried adding some unit tests, but they require a bit more time to do correctly. I propose adding them in a subsequent change so that this PR can make the next release.